jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -755,14 +756,14 @@ def _update_viewer_bound_steps(self, msg={}):
         if not isinstance(viewer.state, ImageViewerState):
             # We round these values to show, e.g., 7.15 instead of 7.1499999
             if hasattr(viewer.state, "x_max") and viewer.state.x_max is not None:
-                bound_step = (viewer.state.x_max) / 100.
-                decimals = -int(np.log10(bound_step))+1
+                bound_step = (viewer.state.x_max - viewer.state.x_min) / 100.
+                decimals = -int(np.log10(bound_step)) + 1

```

Comment:
otherwise this can result in an OverflowError if `bound_step == 0` (not sure what makes sense for the fallback, and will need to apply this to the case for y as well).

```suggestion
                decimals = -int(np.log10(bound_step)) + 1 if bound_step > 0 else 6
```

Reply:
How about just wrapping `bound_step` in `abs()` instead? I think we want that anyway now that I think about it - or are you thinking about a reverse axis case where the up arrow should decrease the value?

Reply:
We still need to handle the `bound_step==0` case in which `log10(0)` will resolve to `-inf` and raise an error when casting to an integer.

Reply:
Honestly we should probably prevent min=max in general somewhere else, but for now I added this check.